gcp_cost_report/main.py

```diff
@@ -185,7 +185,14 @@ def wrap_in_mrkdwn(a):
                 wrap_in_mrkdwn('\n'.join(a[0] for a in all_rows)),
                 wrap_in_mrkdwn('\n'.join(a[1] for a in all_rows)),
             ]
-            blocks = [{'type': 'section', 'fields': body}]
+
+            dashboard_message = {
+                "type": "mrkdwn",
+                "text": "For further details, visit our \
+                <https://datastudio.google.com/reporting/9f134c96-b86c-4a05-9685-4df2438537a4 | GCP Cost Dashboard>",
+            }
+
+            blocks = [{'type': 'section', "text": dashboard_message, 'fields': body}]
```

Comment:
```suggestion
            blocks = [{'type': 'section', 'text': dashboard_message, 'fields': body}]
```

Reply:
Are you sure this works?
![CleanShot 2021-08-05 at 12 55 54@2x](https://user-images.githubusercontent.com/2876012/128283785-d3f604f3-8807-43cf-ac40-81a8ffaf8393.png)

Reply:
Oh, that's weird? I thought it would.
<img width="1156" alt="Screen Shot 2021-08-05 at 1 57 45 pm" src="https://user-images.githubusercontent.com/79084890/128288731-e2dfa15e-7a0e-457a-8c0d-0f472eece473.png">
Is there a simple way for me to test this?

Reply:
What does the error say on your screen?

Reply:
Oh yes, so your example gives `text` a string. I believe you need to pass a dictionary.

Reply:
Ah, sorry for the noise!